app/assets/stylesheets/application.css.scss

```diff
@@ -110,6 +110,30 @@ body {
   background-position: center center;
 }
 
+.application-body-edoras{
+  background-color: black;
+  background-image: url('edoras.jpg');
+  background-size:     cover;
+  background-repeat:   no-repeat;
+  background-position: center center;
+}
+
+.application-body-minas_tirith{
+  background-color: black;
+  background-image: url('minas_tirith.jpg');
+  background-size:     cover;
+  background-repeat:   no-repeat;
+  background-position: center center;
+}
+
+.application-body-mordor{
+  background-color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.